Project.toml

```
@@ -1,6 +1,6 @@
 name = "DataFrames"
 uuid = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
-version = "1.4.0"
+version = "1.4.1"
```

```suggestion
version = "1.4.0"
```
1.4 has not been released yet.

---

Oh whoops, you're right. If you're not planning to release 1.4 soon, should this be backported to another branch and released as a patch version?

---

It is set the version to 1.4, because we want to make a release soon.
(we wanted in for 2 months already but the metadata discussion got super complicated and we are now finishing it)